cmd/session-clear.go

```diff
@@ -44,16 +44,20 @@ var sessionClear = cli.Command{
   
 USAGE:
   {{.HelpName}} SESSION-ID|all
+
+SESSION-ID:
+  SESSION - Session can either be $SESSION-ID
```

Comment:
this seems incomplete. Shouldn't this be `$SESSION-ID` or `all`?

Reply:
yes it should be `all`